docker-compose.yaml

```diff
@@ -31,6 +31,10 @@ services:
       - ANOMALY_OUTBOUND=4
       # Executing Paranoia Level
       # - EXECUTING_PARANOIA=2
+      #
+      # New variable naming
+      - BLOCKING_PARANOIA_LEVEL=1
+      # - DETECTION_PARANOIA_LEVEL=4
 
```

Comment:
Does it make sense to add reporting level here too?

Reply:
Yes, added.